lib/core/test/integration/Test/Integration/Framework/DSL.hs

```diff
@@ -838,28 +850,47 @@ createWalletViaCLI ctx args mnemonics secondFactor passphrase = do
         err <- TIO.hGetContents stderr
         return (c, T.unpack out, err)
 
-deleteWalletViaCLI :: (CmdResult r, HasType Port s) => s -> String -> IO r
-deleteWalletViaCLI ctx walId = cardanoWalletCLI
+deleteWalletViaCLI
+    :: forall t r s. (CmdResult r, KnownCommand t, HasType Port s)
```

Comment:
Maybe we could use a constraint synonym to reduce noise.

Reply:
Yeah, thought of it but some functions have slightly different constraints, and I thought that introducing sometimes an alias, and sometimes not will create more noise than help. In the end, it will mean one other type of constraint to load into our brain when reading :/